### PR TITLE
News entry for PR (13113): "Add Storage dashboard"

### PR DESCRIPTION
**Pull request by bgruening**
This is a test PR that takes a PR description and creates a small news entry for it. The second commit is what the reviewer of this hub PR needs to do manually. The bulk of technical descriptions and images should already be there so that the hub reviewer only needs to clean up and bit and maybe simplify the sentences.

The date is taken from the pr-merge-date, so we can also create these posts retrospectively. There are some really good PR descriptions with (animated) images or even videos, I think they are good content for our general audience that we should re-use in the hub.

Any feedback and comments?

### CODE REVIEW

**Comment by bgruening** [comment]
grafik

File: content/news/2022-03-25_add-storage-dashboard/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Add Storage dashboard from David López
+date: "2022-03-25"
+tease: "New pull request from David López merged: Add Storage dashboard"
+hide_tease: true
+autotoc: false
+tags: ['area/UI-UX', 'kind/feature', 'highlight']
```
Comment (bgruening) [inline content/news/2022-03-25_add-storage-dashboard/index.md:7]: @dannon do you think we can just stick to the Galaxy tags here?

Comment (ElectronicBlueberry) [reply content/news/2022-03-25_add-storage-dashboard/index.md:7]: I think using different tags based on the galaxy tags may be better here. I'd remove the `area/` and `kind/` prefixes, as these seem more like technical details, than what users would search for.

Comment (bgruening) [reply content/news/2022-03-25_add-storage-dashboard/index.md]: Have done so, thanks Laila.

**Pull request merged**